sdk/communication/Azure.Communication.JobRouter/api/Azure.Communication.JobRouter.netstandard2.0.cs

```diff
@@ -387,13 +387,21 @@ public partial class RouterAdministrationClient
         public virtual Azure.Pageable<Azure.Communication.JobRouter.Models.JobQueueItem> GetQueues(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual Azure.AsyncPageable<Azure.Communication.JobRouter.Models.JobQueueItem> GetQueuesAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual Azure.Response<Azure.Communication.JobRouter.Models.ClassificationPolicy> UpdateClassificationPolicy(Azure.Communication.JobRouter.UpdateClassificationPolicyOptions options, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response UpdateClassificationPolicy(string classificationPolicyId, Azure.Core.RequestContent content, Azure.RequestContext context = null) { throw null; }
```

Comment:
There may be a question about whether the classificationPolicyId also lives in the serialized value in the RequestContent.  If it does, which value would take precedence for the update?

Reply:
If the id is provided in the body, server-side there is validation whether the two values match or not. If it does not match, it is returned as 400 BadRequest. If it matches, there is no issue. However, the id must be present in the path. In the body its optional.